### PR TITLE
Adding recentItems route

### PR DESCRIPTION
**Pull request by ahsang**
adds the recent items route as per https://developer.box.com/v2.0/reference#get-recent-items
If this is fine, i can add tests and documentation for this as well

### CODE REVIEW

File: lib/managers/recent-items.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	this.client = client;
+}
+
+/**
+ * Requests all items that have been accessed by a user in the last 90 days or the last 1000 items accessed.
+ *
+ * API Endpoint: '/recent_items'
+ * Method: GET
+ *
+ * @param {Object} [qs] - Additional options can be passed with the request via querystring. Can be left null in most cases.
```
Comment (mattwiller) [inline lib/managers/recent-items.js]: Could you please add documentation for the query string options documented at https://developer.box.com/reference#get-recent-items to the JSDoc here?  You can use the following format:

```
@param {string} [qs.marker] Marker field description goes here
```

**Comment by mattwiller** [comment]
Looks good — thanks again for contributing! 🙌

**Comment by ahsang** [comment]
@mattwiller can you please publish this change on npm too? i am using this for a project so would love it if the feature came in the official build.

**Comment by mattwiller** [comment]
@ahsang Sure!  I'm working on #151 right now, which I would like to get released as well, so once that's finished I'll create a release with both commits in it.

**Comment by mattwiller** [comment]
FYI @ahsang I just released v1.6.0 of the SDK, which includes the Recent Items endpoint.  Thanks again for contributing!

**Comment by ahsang** [comment]
Thanks

**Pull request merged**